main.js

```diff
@@ -95,14 +95,41 @@ for (let index = 0; index < extendProjectBtns.length; index++) {
 
 // client side form validation form
 const form = document.querySelector('#register');
-
 const email = document.querySelector('#email');
-
 const errorMessage = document.querySelector('#error-message');
+const name = document.querySelector('#name');
+const feedback = document.querySelector('#message');
 
 form.addEventListener('submit', (event) => {
   if (email.value !== email.value.toLowerCase()) {
     errorMessage.style.visibility = 'visible';
     event.preventDefault();
   }
 });
+
+// storing data in local Storage
+
+function visitorData() {
+  const visitor = {
+    Name: name.value,
+    Email: email.value,
+    Message: feedback.value,
+  };
+  // store the object
+  localStorage.setItem('visitor', JSON.stringify(visitor));
+}
+
+// put focus out fonction in function visitor data
+// [name, email, feedback].forEach((input) => input.addEventListener('focusout', visitorData));
```

Comment:
- [ ] @team its not good practise to have commented code in your codebase, kindly remove the commented code.

Reply:
Thanks it is fixed